snippets/ztp_example-sno.yaml

```diff
@@ -1,6 +1,6 @@
 # example-node1-bmh-secret & assisted-deployment-pull-secret need to be created under same namespace example-sno
 ---
-apiVersion: ran.openshift.io/v1
+apiVersion: ran.openshift.io/v2
```

Comment:
Has it been agreed that siteconfigv2 will be released in 4.16?

Reply:
@irinamihai We must use v1 and not v2. QE only tests and verifies v1. It seems our upstream examples were changed to v2 at some point, we need to change that back to v1 in our example.